client/FxAccountClient.js

```diff
@@ -80,6 +80,18 @@ define([
    *     @param {String} type Type of device (mobile|desktop)
    *     @param {string} [callback] Device's push endpoint.
    *     @param {string} [publicKey] Public key used to encrypt push messages.
+   *   @param {Object} [options.metricsContext={}] Metrics context metadata
+   *     @param {String} [flowId] identifier for the current event flow
+   *     @param {Number} [flowBeginTime] flow.begin event time
+   *     @param {String} [context] context identifier
+   *     @param {String} [entrypoint] entrypoint identifier
+   *     @param {String} [migration] migration identifier
+   *     @param {String} [service] service identifier
+   *     @param {String} [utm_campaign] marketing campaign identifier
+   *     @param {String} [utm_content] marketing campaign content identifier
+   *     @param {String} [utm_medium] marketing campaign medium
+   *     @param {String} [utm_source] marketing campaign source
+   *     @param {String} [utm_term] marketing campaign search term
```

Comment:
I'm not sure if there is a way to make proper besides doing something like `options.metricsContext.flowId`, need to check the docs.